samples/tfm/tfm_secure_peripheral/secure_peripheral_partition/secure_peripheral_partition.c

```diff
@@ -22,21 +22,56 @@
 #include <zephyr/autoconf.h>
 #include <zephyr/devicetree.h>
 
-#define BUTTON_PIN      DT_GPIO_PIN(DT_NODELABEL(button0), gpios)
-
-#if defined(CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP_NS)
-#define SCK_PIN         47 /* P1.15 */
-#define MOSI_PIN        45 /* P1.13 */
-#elif defined(CONFIG_SOC_SERIES_NRF91X) && defined(CONFIG_BUILD_WITH_TFM)
-#define SCK_PIN         13 /* P0.13 */
-#define MOSI_PIN        11 /* P0.11 */
+#include <hal/nrf_gpio.h>
+#include <nrfx_gpiote.h>
+#include <hal/nrf_gpio.h>
+#include <hal/nrf_gpiote.h>
+
+
+#if defined(CONFIG_SOC_SERIES_NRF53X)
+#define BUTTON_PIN	NRF_PIN_PORT_TO_PIN_NUMBER(23,0)  /* P0.23 */
```

Comment:
this is horrible, get it from dts...

Reply:
I couldn't find a way to get the port and the pin from DTS. Do you have a suggestion? The DT_GPIO_PIN which was used before just return the pin number and not the port as well but this only works (by accident)  with port 0.

Reply:
There are macros in https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/nordic/common/soc_nrf_common.h which can be used

Reply:
Ok cool, I didn't know about these. I updated the define

Reply:
@nordicjm please have a second look